templates/cspell-templates.txt

```diff
@@ -22,6 +22,7 @@ eastus
 envconfig
 ezfunc
 fasterxml
+funcapp
```

Comment:
is this the official term?  Looks like we might be using the implementation label for our comments, which might not be OK.  I think we should refer to `funcapp` as `Azure Functions` (the commercial name).

I see that we use `functionapp` for bicep, and it is already within the dictionary (cli/azd/.vscode/cspell-azd-dictionary.txt).  So, should we use that instead?   (for bicep comments)?

Reply:
This is completely incorrect, as you pointed out. It should be `function`. I missed out on what the official term is.

Reply:
Removed